optuna/terminator/terminator.py

```diff
@@ -101,8 +100,8 @@ class Terminator(BaseTerminator):
 
     def __init__(
         self,
-        improvement_evaluator: Optional[BaseImprovementEvaluator] = None,
-        error_evaluator: Optional[BaseErrorEvaluator] = None,
+        improvement_evaluator: BaseImprovementEvaluator | None = None,
+        error_evaluator: BaseErrorEvaluator | None = None,
         min_n_trials: int = DEFAULT_MIN_N_TRIALS,
     ) -> None:
```

Comment:
```suggestion
    ):
```

Reply:
[PEP 484](https://peps.python.org/pep-0484/#the-meaning-of-annotations) says:

> the return type of `__init__` ought to be annotated with `-> None`.